tests_e2e/playwright/conftest.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import os
+
+import dotenv
+import pytest
+from navigator import Navigator
+
+logger = logging.getLogger()
+
+
+@pytest.fixture(scope="session")
+def _navigator_session(browser_name):
+    """Set up a navigator instance, login with username/password, start
+    a server. Teardown when session is complete.
+    Do not use this for individual tests, use `navigator` fixture
+    for tests."""
+    dotenv.load_dotenv()
+    # try/except added here in attempt to reach teardown after error in
+    # order to close the browser context which will save the video so I debug
+    # the error.
+    try:
+        nav = Navigator(
+            nebari_url=os.environ["NEBARI_FULL_URL"],
+            username=os.environ["USERNAME"],
+            password=os.environ["PASSWORD"],
+            headless=False,
+            browser=browser_name,
+            auth="password",
+            instance_name=os.environ["INSTANCE_NAME"],
+            video_dir="videos/",
+        )
+        nav.login_password()
+        nav.start_server()
+        yield nav
```

Comment:
What if we fail to create the `Navigator`? We should oust this outside the `try / except` or otherwise the `nav.teardown()` fails as well.